### PR TITLE
Add the initial content of proposals

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Add the initial content of the proposals.

- Any checklist items to be verified before production deployment?
- Special styling concerns? 
- Related media files?

### CODE REVIEW

File: website/two-column-pages/docs/season-of-docs-2019.md [hunk 1]
```diff
@@ -1,762 +1,144 @@
...
-### Description
-
-The WSO2 API Microgateway has an immutable runtime. Meaning that if an update is made to the interface or any other configuration the microgateway runtime needs to be rebuilt and redeployed. This is not a huge concern for Kubernetes like environments where the infrastructure itself handles this via rolling updates. However for traditional VM based infrastructures it is important to build a mechanism to support graceful updates. When this is implemented, we should have a model where a config/interface change can be deployed on the Microgateway with no downtime.
-
-### Deliverables
-
-A framework for gracefully updating the Microgateway
-
-### Skills Needed
+If you are interested in working on any of the below projects and if you need further information, please feel free to contact the mentors listed under each proposal below or write to us via ballerina-dev@googlegroups.com or via Slack.
```
Comment (nadeeshaan) [inline website/two-column-pages/docs/season-of-docs-2019.md]: Shall we add the slack channel's link as well?

File: website/two-column-pages/docs/season-of-docs-2019.md [hunk 2]
```diff
@@ -1,762 +1,144 @@
...
 ### Deliverables
 
-Prometheus alerting based artifacts, Autoscaling policies for pod autoscaling based on Prometheus
+* A step-by-step guide, which will go to the Ballerina website showcasing the capabilities of the given set of Ballerina Tools based on a real-world integration use case.
+* A blog expressing your experience and the learning gathered throughout the project and through the engagement with the Ballerina community.
 
 ### Skills Needed
 
-Kubernetes, Docker, Prometheus
+Ballerina, Visual Studio Code, Markdown
```
Comment (nadeeshaan) [inline website/two-column-pages/docs/season-of-docs-2019.md]: Shall we mark Visual Studio Code not mandatory?

File: website/two-column-pages/docs/season-of-docs-2019.md (hunk 1 above)
Comment (praneesha) [reply website/two-column-pages/docs/season-of-docs-2019.md]: Done

File: website/two-column-pages/docs/season-of-docs-2019.md (hunk 2 above)
Comment (praneesha) [reply website/two-column-pages/docs/season-of-docs-2019.md]: Done

**Pull request merged**